helm/robusta/values.yaml

```diff
@@ -586,7 +586,7 @@ image:
 
 # parameters for the robusta forwarder deployment
 kubewatch:
-  image: ~  # image can be used to override image.registry/imageName
+  image: ~ # image can be used to override image.registry/imageName
   imageName: kubewatch:v2.7.0
```

Comment:
Don't we need a new Kubewatch image?

Reply:
We need to change it after new kubewatch release